src/Chr.Avro.Binary/BinarySerializerBuilder.cs

```diff
@@ -1610,19 +1610,26 @@ public override Delegate BuildDelegate(TypeResolution resolution, Schema schema,
 
             if (source != target)
             {
-                if (source == typeof(DateTime) || source == typeof(DateTimeOffset))
+                if (source == typeof(DateTime))
                 {
-                    if (source == typeof(DateTimeOffset))
-                    {
-                        result = Expression.PropertyOrField(result, nameof(DateTimeOffset.UtcDateTime));
-                    }
-
-                    var convertDateTime = typeof(DateTime)
+                    var convertDateTimeOffset = typeof(DateTime)
```

Comment:
convertDateTIme would be less confusing?

Reply:
good catch, typo on my part